test/integration/framework/BUILD.bazel

```diff
@@ -6,12 +6,6 @@ go_library(
         "apiserver.go",
         "helpers.go",
     ],
-    data = [
-        "//deploy/crds:crds.yaml",
-        "//hack/bin:com_coreos_etcd",
-        "//hack/bin:io_kubernetes_kube-apiserver",
-        "//hack/bin:kubectl",
-    ],
```

Comment:
moved these dependencies to the correct files (the files that load these dependencies from the file-system)